src/scss/components/menu.scss

```
@@ -0,0 +1,158 @@
+  }
+
+  li {
+    border-bottom: 1px solid hsla(0, 0%, 79%, .5);
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    position: relative;
+
+    &:last-child,
+    &:last-of-type {
+      border-bottom: none;
+    }
+
+    &.has-icon {
+      > a,
+      > span:not([class*="fa"]) {
+         padding-left: 3.1rem;
+      }
+    }
+
+    .fa-li {
+      left:  .9rem;
+    }
+
+    .fas {
+      color: var(--uids-grey-cool);
+    }
+
+    // Active menu item has a grey indicator.
+    &.is-active {
+      a {
+        &:after {
+          height: var(--menu-link-indicator-height);
+          width: var(--menu-link-indicator-width);
+          transform: var(--menu-link-transform-end);
+        }
+      }
+    }
+
+    a,
+    span:not([class*="fa"]) {
+      color: var(--menu-link-color);
+      display: block;
+      padding: .9rem 1.5rem;
+      position: relative;
+      text-decoration: none;
+
+      &:focus,
+      &:hover {
+        text-decoration: underline;
+      }
+
+      // Menu link indicator base definition.
+      // Adjusted on hover and when the parent li has the .is-active class.
+      &:after {
+        background: var(--uids-grey-light);
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 0;
+        height: 0;
+        content: "";
+        transform-origin: center;
+        transform: var(--menu-link-transform-start);
+        transition: 0.3s all ease-in-out;
+        -webkit-transition: 0.3s all ease-in-out;
+      }
+
+      // Adjustments for light colored backgrounds.
+      [class*="bg--white"] &,
+      [class*="bg--"] [class*="bg--white"] &,
+      [class*="bg--gray"] &,
+      [class*="bg--"] [class*="bg--gray"] & {
+        color: variables.$dark-gray;
+      }
+    }
+
+    // Nested menus.
+    & > ul.menu {
+      // Indent nested menus.
+      margin-left: 1.7rem;
+
+      // Remove bottom border from nested list items.
+      li {
+        border-bottom: none;
+
+        // Reduce vertical padding for nested menu items.
+        a,
+        span:not([class*="fa"]) {
+          padding-block-start: .5rem;
+          padding-block-end: .5rem;
+        }
+      }
+    }
+  }
+
+  // Adds an underline on hover.
+  > li > a:hover:after {
+    transform: var(--menu-link-transform-end);
+    background: var(--brand-primary);
+    height: var(--menu-link-indicator-height);
+    width: var(--menu-link-indicator-width);
+  }
+}
+
+.menu--horizontal {
+  @include utilities.container-query() {
+    --menu-link-indicator-width: 100%;
+    --menu-link-indicator-height: 5px;
+    --menu-flex-direction: row;
+    --menu-link-transform-start: scaleX(0%);
+    --menu-link-transform-end: scaleX(100%);
+    display: flex;
+    flex-wrap: wrap;
+
+    // Remove border between horizontal menu items.
+    li {
+      border-bottom: none;
+    }
+
+    // Hide nested menus in horizontal mode for now.
+    // @todo: Figure out how to display nested menus on hover.
```

```suggestion
    // @todo: Figure out how to display nested menus on hover and
    //   keep in accordance with https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html.
```

Some of the recent training/reading I've done recommends against expand on hover in menus for how easy it can be to fail this requirement and create accessibility issues. I think our current menus are rather accessible in this respect, with the slight delay before expansion (to avoid accidental opening and obscuring of content) and clear indicators of their use. So I think we can and should pursue it, but wanted to add a specific reference for that.

---

I think our intention is to pursue using this library: https://accessible-menu.dev/